src/main/java/com/videoclub/suriken/model/Movie.java

```diff
@@ -0,0 +1,148 @@
+package com.videoclub.suriken.model;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Entity
+public class Movie {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private Long movId;
+
+    private String movName;
+
+    private int movYear;
+
+    @Enumerated(EnumType.STRING)
+    private Genre movGenre;
+
+    private String movDirector;
+
+    private int stock;
+
+    @ManyToMany()
```

Comment:
Pitanje za intervju: Sta je cascade i sta je fetch type kod relacija u hibernate?
Malo teze pitanje: Sta je n+1 problem u hibernate? (Jaaako popularno pitanje) [Odgovor](https://vladmihalcea.com/n-plus-1-query-problem/#:~:text=The%20N%2B1%20query%20problem%20happens%20when%20the%20data%20access,the%20larger%20the%20performance%20impact.)